### DISCUSSION TITLE
using other lr_scheduler like ReduceLROnPlateau

### DISCUSSION THREAD

**Discussion by mzalaki00** [facebookresearch/detectron2#3548] [Q&A]
how to use torch learning rate scheduler and connect it to detectron2 config and model.
it seems detectron2 support just two predefined lr_scheduler like warmuoCosineLr and multiStepLr.
how to use ReduceLROnPlateau in detectron2 ??
please give solution with code example

**Comment by JasspalSingh** [comment]
did you got any solution?

**Reply by mzalaki00** [reply]
not yet @JasspalSingh
